### ISSUE TITLE
Minerva calculation is wrong with salt city

### ISSUE THREAD

**Issue by dmk11** [yav/concordia#34]

**Comment by yav** [comment]
I think it might be fixed now.  Also the way the points on the Minerva market cards is supposed to work is that it should show the points as if you already own the card (i.e., if it is more valuable than you other specialists it should count salt for it, but if it is not, then it will not count the salt for it)

**Comment by yav** [comment]
I am not sure how the split salt could happen, I wonder if we made a mistake when looking at the board?

**Comment by dmk11** [comment]
In my case, I already own Smith and Farmer is showing as if it's counting my salt cities even though those 2 minerva is worth the same.

**Comment by dmk11** [comment]
I dont think we made a mistake looking at board.  You can still see the board, actually: http://164.90.148.49:9002/?player=Dian 
Also, when do you do the minerva calculation, it would be nice if you say (2+1)x3=12 instead of what currently is showing as 1x12 = 12 (in the case that I had 2 smith & 1 salt cities), it'll be easier to debug where your calculation error comes in too.

**Comment by yav** [comment]
I like the idea of showing it separately.

I think the problem was that the salt was assigned to the least valuable cities rather than the most ones (order was Brick, Wheat, Smith, Wine, Cloth, but it should be exactly the other way around)  

In the example above, I think you have: 2 wheat cities, 3 tool cities, and 2 salt cities, so it is computing
  Farmer Minerva: 2 wheat + 2 salt = 4 * 3 = 12
  Smith Minerva: 3 tool = 3 * 3 = 9
